gefserver/main.go

```diff
@@ -31,7 +31,7 @@ func main() {
 	defer d.Close()
 
 	var p *pier.Pier
-	p, err = pier.NewPier(&d, config.TmpDir)
+	p, err = pier.NewPier(&d, config.TmpDir, config.Timeouts.JobExecution, config.Timeouts.CheckInterval)
```

Comment:
It's too fine grained to send all timeouts individually, let's store the entire timeout structure into pier